src/main/java/org/folio/circulation/rules/Text2Drools.java

```diff
@@ -158,6 +158,16 @@ private void setIndentation(TerminalNode node) {
     indentation = Integer.parseInt(node.getText().substring(6));
   }
 
+  @Override
+  public void enterSimpleStatement(CirculationRulesParser.SimpleStatementContext ctx) {
+    ExprContext expr = ctx.expr();
+    Token token = ctx.getStart();
+    if (expr.criterium() != null && expr.policies() == null) {
+      throw new org.folio.circulation.rules.CirculationRulesException(
+        "Policy missing.", token.getLine(), token.getCharPositionInLine() + 1);
```

Comment:
Which policy is missing? Wouldn't it be clearer to the end user if we just said that ':' symbol is missing?

Reply:
I followed my ticket

Expected Results:
Syntax error message 'policy missing'

But of course I don't insist, The message is changed